### PR TITLE
ceph: update object prefixed manifests and related docs

### PR DESCRIPTION
**Pull request by subhamkrai**
apply latest changes to prefixed manifests and
related docs.

Signed-off-by: subhamkrai <subhamkumarrai03@gmail.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

### CODE REVIEW

File: cluster/examples/kubernetes/ceph/object-test.yaml [hunk 1]
```diff
@@ -20,5 +20,5 @@ spec:
   gateway:
     type: s3
     port: 80
-    securePort:
+    # securePort:
```
Comment (satoru-takeuchi) [inline cluster/examples/kubernetes/ceph/object-test.yaml:23]: It's better to leave a comment that means what is the latest changes. I guess you mean the following commit.

https://github.com/rook/rook/commit/e21854145af45545b6282b5066e4018044c69382

```suggestion
    # securePort: 443
```

How about setting the default value to `securePort`. Then users can ebale securePort without setting it explicitly. In addition, could you also change the similar `securePort:` fields of the following files?

https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/object-multisite.yaml#L47
https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/object-multisite-pull-realm.yaml#L61
https://github.com/rook/rook/blob/master/Documentation/ceph-object-store-crd.md
https://github.com/rook/rook/blob/master/Documentation/ceph-object.md

Comment (subhamkrai) [reply cluster/examples/kubernetes/ceph/object-test.yaml]: Done.

**Review by satoru-takeuchi — APPROVED** [review]

File: cluster/examples/kubernetes/ceph/object-test.yaml (hunk 1 above)
Comment (leseb) [reply cluster/examples/kubernetes/ceph/object-test.yaml]: Just setting the port won't be enough for a secured connection but still, it's better to give it a value.

**Comment by subhamkrai** [comment]
> Does Helm need to be updated too?

I'm not sure about Helm but IMO, according to [this](https://github.com/rook/rook/pull/5986) I have updated files that are related and needed to update.

**Comment by satoru-takeuchi** [comment]
I confirmed that it's not necessary to update Helm files.

**Review by leseb — APPROVED** [review]

**Pull request merged**